packages/website/pages/stats.js

```diff
@@ -11,18 +11,20 @@ import { abbreviateNumber } from 'js-abbreviation-number'
  * @returns {{ props: import('../components/types.js').LayoutProps}}
  */
 export function getStaticProps() {
-  const logos = fs.readdirSync('public/images/marketplace-logos')
+  const logos = fs.readdirSync('public/images/marketplace-logos/home')
```

Comment:
![image](https://user-images.githubusercontent.com/954596/157568318-c498f74e-264a-4664-81f7-f3f0ce17f599.png)
this wont break but might tweak this tiny thing, I won't block but heads up

Reply:
Otherwise its all showking up just fine here:
https://38990119.nft-storage-1at.pages.dev/stats/
![image](https://user-images.githubusercontent.com/954596/157568567-f536da0d-4f4e-4c80-97d6-cbdf24c107cf.png)